client/src/app/components/profile/profile.component.html

```diff
@@ -1,8 +1,18 @@
-<h2 class="page-header">Profile Page</h2>
+<div class="container-fluid">
+		<ul class="nav navbar-nav navbar-left">
+			<li><h2 class="page-header">Profile Page</h2></li>
+		</ul>
+		<ul class="nav navbar-nav navbar-right">
+			<li><button type="button" class="btn toRight" [routerLink]="['/edit-profile/']">Edit</button></li>
+		</ul>		
+</div>
+	
```

Comment:
i think we need some linting and indentation set up in our project to avoid spacing issue